xAPISGProcessor.ipynb

```diff
@@ -74,10 +74,25 @@
     "import ipywidgets as widgets\n",
     "from IPython.display import display, clear_output\n",
     "import pprint\n",
-    "plt.style.use('dark_background')\n",
-    "plt.rcParams['figure.figsize']=[10, 8]"
+    "plt.style.use('dark_background')"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "! jupyter nbextension enable --py --sys-prefix widgetsnbextension\n",
+    "! jupyter nbextension install --py --symlink --sys-prefix ipympl\n",
+    "! jupyter nbextension enable ipympl --py --sys-prefix"
```

Comment:
Is there another way to do this? Seems like too low level code to have it in the main notebook.